mautic/mautic
# [UI] Tooltips

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR is based [on another PR](https://github.com/mautic/mautic/pull/13735), where I introduce tokens to replace most of the UI variables, aiming to improve consistency in all components. **We need it to be merged before this one becomes available for code review.**

This PR increases the font size for small elements and increases the padding of tooltips.




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/mautic/mautic/assets/116097999/b3b45e7c-2b12-44fe-afb0-623ad78a5342)                                      | ![image](https://github.com/mautic/mautic/assets/116097999/7db466af-2575-4a4c-8dad-132605a44175)



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Account > Look at the descriptions below the titles, they're 8% bigger
3. Open Contacts > Create new > Hover over any field with a tooltip
